lib/flex/commerce/api/version.rb

```diff
@@ -1,7 +1,7 @@
 module Flex
   module Commerce
     module Api
-      VERSION = "0.3.1"
+      VERSION = "0.3.31"
```

Comment:
If you are making any more changes, please remove this file - Ive been meaning to do it for ages